document/js-api/index.bs

```diff
@@ -446,22 +446,22 @@ enum ImportExportKind {
 };
 
 dictionary ModuleExportDescriptor {
-  required USVString name;
+  required DOMString name;
   required ImportExportKind kind;
   // Note: Other fields such as signature may be added in the future.
 };
 
 dictionary ModuleImportDescriptor {
-  required USVString module;
-  required USVString name;
+  required DOMString module;
+  required DOMString name;
   required ImportExportKind kind;
```

Comment:
These are only used for values that the JS engine constructs, so might as well keep USVString for clarity.